integrations/aws/config.yaml

```diff
@@ -0,0 +1,14 @@
+port:
+  clientId: '{{ from env PORT_CLIENT_ID }}'
+  clientSecret: '{{ from env PORT_CLIENT_SECRET }}'
+eventListener:
+  type: POLLING
+integration:
+  identifier: 'my-aws-integration'
+  type: 'aws'
+  config:
+    awsAccessKeyId: '{{ from env AWS_ACCESS_KEY_ID }}'
+    awsSecretAccessKey: '{{ from env AWS_SECRET_ACCESS_KEY }}'
+    awsRealTimeUpdatesRequestsApiKey: '{{ from env AWS_API_KEY }}'
+    organizationRoleArn: '{{ from env ORGANIZATION_ROLE_ARN }}'
+    accountReadRoleName: '{{ from env ACCOUNT_READ_ROLE_NAME }}'
```

Comment:
make sure this is updated only with the required parameters

Reply:
All of these parameters are optional, can you elaborate?
I didn't really understand...